pkgs/development/python-modules/pyflick/default.nix

```diff
@@ -1,27 +1,36 @@
 {
   lib,
-  buildPythonPackage,
-  pythonOlder,
-  fetchPypi,
   aiohttp,
+  buildPythonPackage,
+  fetchFromGitHub,
+  json-api-doc,
   python-dateutil,
+  setuptools-scm,
+  setuptools,
 }:
 
 buildPythonPackage rec {
-  pname = "pyflick";
-  version = "0.0.2";
-  format = "setuptools";
-
-  disabled = pythonOlder "3.6";
+  pname = "py-flick";
```

Comment:
https://github.com/ZephireNZ/PyFlick/blob/v1.1.2/pyproject.toml#L6

That should be pyflick.